main/src/mill/main/MainModule.scala

```diff
@@ -173,4 +179,34 @@ trait MainModule extends mill.Module{
       }
     }
   }
+
+  /**
+    * Deletes the given targets from the out directory. Providing no targets will clean everything.
+    */
+  def clean(targets: String*) = mill.T.command {
+    val result = Try {
```

Comment:
No idea why you are wrapping this in a `Try`; you just need to return a simple `Result` object

Reply:
The ammonite functions call things which can throw exceptions. Is it acceptable to let those exceptions be thrown?